libraries/CMakeLists.txt

```diff
@@ -13,6 +13,7 @@ set( LIBRARIES
    lib-preferences
    lib-math
    lib-files
+   lib-ipc
```

Comment:
This line should be added in the same commit that defines the source code files in lib-pic.  Not later.